app/src/main/java/io/gnosis/safe/helpers/AddressTooltip.kt

```diff
@@ -0,0 +1,41 @@
+package io.gnosis.safe.helpers
+
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.graphics.drawable.ColorDrawable
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.StyleSpan
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.PopupWindow
+import android.widget.TextView
+import io.gnosis.safe.R
+
+class AddressTooltip(
+    val context: Context,
+    address: String
+) : PopupWindow(context) {
+
+    init {
+        setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+        isOutsideTouchable = true
+        isFocusable = false
+
+        contentView = LayoutInflater.from(context).inflate(R.layout.popup_address_tooltip, null)
+
+        //make first & last 4 characters bold
+        val addressString=  SpannableStringBuilder(address.substring(0, address.length / 2) + "\n" + address.substring(address.length / 2, address.length))
```

Comment:
Should this be a utils (we will use this in many places, right?)

Reply:
As a matter of fact we do have a utils for this 🤦 I will replace with it.